### ISSUE TITLE
error creating smtp client: EOF

### ISSUE THREAD

**Issue by camps7ee** [shenxn/protonmail-bridge-docker#42]
I'm trying to connect other services (non email-clients) to use the pm-bridge as an SMTP (alert notifications etc.) and I just keep getting this error:

`error getting SMTP client: error creating smtp client: EOF`

If I look at a container logs, I can see that the container is purposefully rejecting the connection requests, and I cannot for the life of me figure out why :(

`today at 01:39:062021/12/11 00:39:06 socat[1379] E connect(5, AF=2 127.0.0.1:1143, 16): Connection refused
today at 01:41:342021/12/11 00:41:34 socat[1380] E connect(5, AF=2 127.0.0.1:1025, 16): Connection refused`

Ports 1025 and 1143 are open on the host as well as on my router (just in case) 

Connection works fine as long as I use Thunderbird/AppleMail/Outlook........I've seen that others have gotten their Nextcloud/Bitwarden/etc. services to use pm-bridge as SMTP. Would someone mind sharing their magic?

Thanks a million in advance!

/C

**Comment by camps7ee** [comment]
Found the problem:

-  Whenever `./entrypoint.sh` is exited (is that a word? lol) ports 1025 / 1143 get closed off, I'm suspecting that this has something to do with socat maybe?

**Solution:**
Once your bridge is up and running and configured via `protonmail-bridge --cli` exit the CLI and just run `./entrypoint.sh` from there, just close down that terminal session, without leaving the script, whenever you leave the script all its services stop running.

I wish I'd have a merry christmas now, instead, I have a new issue:
`error enabling StartTLS: x509: certificate is valid for 127.0.0.1, not 192.168.10.101`

At least I got a new issue now so thats good :)
